.changeset/sixty-cherries-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/eslint-plugin': patch
+---
+
+eslint autofix for mui ThemeProvider
```

Comment:
I think we should add autofix to the Vale rules here as part of this PR: https://github.com/backstage/backstage/blob/master/.github/vale/config/vocabularies/Backstage/accept.txt

Or you can put backticks (`) around it

Reply:
Thanks, feel the same 👍